test/unit/wifi.js

```diff
@@ -90,13 +92,66 @@ exports['Tessel.prototype.findAvailableNetworks'] = {
             Encryption: WPA2 PSK (CCMP)
 
 `;
+    // Do not remove the blank line at the end of preceding string!!
+
+    this.tessel.findAvailableNetworks()
+      .then((found) => {
+        test.equal(found.length, 3);
+        test.equal(this.findAvailableNetworks.callCount, 1);
+        test.equal(found[0].ssid, 'best');
+        test.equal(found[1].ssid, 'middle');
+        test.equal(found[2].ssid, 'worst');
+        test.done();
+      });
+
+    this.tessel._rps.stdout.push(networks);
+
+    setImmediate(() => {
+      this.tessel._rps.emit('close');
+    });
+  },
```

Comment:
Not sure why github is crediting me for this—I think the diff is just wonky, because my changes below look like a modification instead of an addition